schedule.html

```diff
@@ -44,152 +45,161 @@
-                        <td>10:00am - 10:30am</td>
-                    </tr>
-
-                    <tr>
-                        <td>Game</td>
-                        <td>11:00am - 11:30am</td>
-                    </tr>
-
-                    <tr>
-                        <td>Lunch (Dairy)</td>
-                        <td>12:30pm - 1:15pm</td>
-                    </tr>
-
-                    <tr>
-                        <td>Stop coding</td>
-                        <td>1:30pm</td>
-                    </tr>
-
-                </tbody>
-
-                <thead>
-                    <tr>
-                        <th scope="col">Conclusion</th>
-                        <th scope="col">1:30pm - 3:10pm</th>
-                    </tr>
-                </thead>
-
-                <tbody>
-
-                    <tr>
-                        <td>Judges walk around</td>
-                        <td>1:30pm - 2:00pm</td>
-                    </tr>
-
-                    <tr>
-                        <td>Judges deliberate</td>
-                        <td>2:00pm - 2:10pm</td>
-                    </tr>
-
-                    <tr>
-                        <td>Finalists Present</td>
-                        <td>2:15m - 2:40pm</td>
-                    </tr>
-
-                    <tr>
-                        <td>Final deliberation</td>
-                        <td>2:40pm - 2:50pm</td>
-                    </tr>
-
-                    <tr>
-                        <td>Closing remarks, winner announced</td>
-                        <td>2:40pm - 3:10pm</td>
-                    </tr>
-
-                </tbody>
-            </table>
-<!--        </div>-->
-    </div>
-    <br>
-    <br>
+<br>
+<div class="HackYU-Schedule">
+    <!--        <div class="schedule-column">-->
+    <table class="table table-hover">
+        <thead>
+        <tr>
+            <th scope="col">Kick-Off</th>
+            <th scope="col">9:30pm - 10:20pm</th>
+        </tr>
+        </thead>
+
+        <tbody>
+
+        <tr>
+            <td>Sign-in</td>
+            <td>9:30pm - 10:00 pm</td>
+        </tr>
+
+        <tr>
+            <td>Opening remarks</td>
+            <td>10:00pm - 10:20pm</td>
+        </tr>
+
+        </tbody>
+
+        <thead>
+        <tr>
+            <th scope="col">Coding</th>
+            <th scope="col">10:30pm - 1:30pm</th>
+        </tr>
+        </thead>
+
+        <tbody>
+
+        <tr>
+            <td>Workshop #1: <br/>
+                <div class="workshop-details">Noam Annenberg,</div>
+                <div class="workshop-details">"Building a Full Stack Web App"</div></td>
+            <td>10:45pm - 11:15pm</td>
+        </tr>
+
+        <tr>
+            <td>Workshop #2: <br/>
+                <div class="workshop-details"> Yosef Sklar,</div>
+                <div class="workshop-details">"From HWs to Hackathons:
+                    How what you’re learning in school translates to web applications"</div></td>
```

Comment:
```suggestion
                <div class="workshop-details">"How to Get from CS Classes to Web Applications"</div></td>
```

This is a shorter title that I found on the schedule google doc